docs/docs/general/05-Integrations/containers.md

```diff
@@ -104,16 +114,26 @@ export { aui };
 
 ```
 
-## Connect via VNC
+## Tag Naming Schema
+We use the following tag schema:
+
+```
+<controller-version>-<browser-name>-<browser-version>-amd64
 
-To check what is happening inside a running container, you can connect via VNC. For this, you need a VNC client like [Remmina](https://remmina.org/). When starting the [Docker Container manually](#starting-container-manually), you have to map the internal port `5900` to a free port on your machine that you, then, can connect to. When [using the testcontainers example code](#starting-container-from-within-beforeall-using-testcontainers), the VNC port to connect to is logged to the console.
+# browser-name: chrome only for now
 
-When connecting, enter the password `askui` when asked.
+# Example
+0.11.2-chrome-119.0.6045.123-amd64
+```
 
-Example of a VNC connection with a Chrome browser running inside a container:
+> ❗️ `<controller-version>` is _NOT_ the AskUI version ❗️
```

Comment:
I would add a link to `AskUI` and rename it to "`askui` nodejs lib" so that this is absolutely clear.

Reply:
Great suggestion!

Not sure what the target of the link should be, to be honest ❓